public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
```

Comment:
Inline style is a bad idea. Please consider moving these inline-styles to the scss file